### DISCUSSION TITLE
Can not install delegates for imagemagick

### DISCUSSION THREAD

**Discussion by avkabanov** [ImageMagick/ImageMagick#2762] [Help]
I'm installing imagemagick from the source. 
OS: Debian, IM version:  7.0.10-34

cwebp is installed, version 1.1.0

but when I run `./configure --with-webp=yes` it gives me: 
`WEBP              --with-webp=yes             no`, 

I've checked config.log - I can not find any error. I've spent two days trying to solve this! What I'm doing wrong?

**Comment by snibgo** [comment]
"--with-webp=yes" will try to install webp as in internal delegate. For this, it needs the webp library libwebp. The presence or absence of the executable cwebp is not relevant.

**Reply by avkabanov** [reply]
Thanks for your answer! I'm a bit lost with libwebp libraries, for example here: 
https://launchpad.net/ubuntu/+source/libwebp and here
https://imagemagick.org/download/delegates/ I see that the latest version of libwebp  is 0.6.x, but here 
https://storage.googleapis.com/downloads.webmproject.org/releases/webp/index.html I see that 1.x version has been released more that 2 years ago. Is it that difference in a version or is this different libraries?

**Comment by snibgo** [comment]
Sorry, I know nothing about webp libraries. With a pre-built binary of IM v7.0.10-27 on Windows 8.1, I get:
```
magick -list format | grep -i webp

     WEBP* WEBP      rw+   WebP Image Format (libwebp 1.0.3 [020E])
```
